client/src/stylesheets/_base.scss

```diff
@@ -75,3 +76,13 @@ body {
         text-decoration: underline;
     }
 }
+.container-block {
+    position: relative;
+    width: 1170px;
```

Comment:
Don't use fixed widths.
If we go with 100% or 100vw, it will help us make responsive content.